lib/runtime_tools/doc/src/scheduler.xml

```diff
@@ -35,12 +35,53 @@
   <module since="OTP 21.0">scheduler</module>
   <modulesummary>Measure scheduler utilization</modulesummary>
   <description>
-    <p>This module contains utility functions for easier measurement and
-    calculation of scheduler utilization, otherwise obtained from calling the
-    more primitive <seeerl marker="erts:erlang#statistics_scheduler_wall_time">
-    <c>statistics(scheduler_wall_time)</c></seeerl>.</p>
-    <p>The simplest usage is to call <seemfa marker="#utilization/1">
-    <c>scheduler:utilization(Seconds)</c></seemfa>.</p>
+    <p>
+      This module contains utility functions for easy measurement and
+      calculation of scheduler utilization.
+    </p>
+    <p>
+      The simplest usage is to call the blocking
+      <seemfa marker="#utilization/1"><c>scheduler:utilization(Seconds)</c></seemfa>.
+      For non blocking and/or continuous calculation of scheduler utilization,
+      the recommended usage is:
+    </p>
+    <list>
+      <item><p>
+	First call <seeerl marker="erts:erlang#system_flag_scheduler_wall_time">
+	<c>erlang:system_flag(scheduler_wall_time,true)</c></seeerl> to enable
+	scheduler wall time measurements.
+      </p></item>
+      <item><p>
+	Call <seemfa marker="#get_sample/0"><c>get_sample/0</c></seemfa> to
+	collect samples with some time in between.
+      </p></item>
+      <item><p>
+	Call <seemfa marker="#utilization/2"><c>utilization/2</c></seemfa> to
+	calculate the scheduler utilization in the interval between two
+	samples.
+      </p></item>
+      <item><p>
+	When done call
+	<seeerl marker="erts:erlang#system_flag_scheduler_wall_time">
+	<c>erlang:system_flag(scheduler_wall_time,false)</c></seeerl> to disable
+	scheduler wall time measurements and avoid unecessary cpu overhead.
+      </p></item>
+      </list>
+      <p>
+	Note scheduler wall time measurements will automatically be disabled if
+	the process that called
+	<seeerl marker="erts:erlang#system_flag_scheduler_wall_time">
+	<c>erlang:system_flag(scheduler_wall_time,true)</c></seeerl> terminates.
```

Comment:
It might be good, here or at some other point in this introduction, to explain what the consequences of the flag being set to false between samples is -- specifically, that an utilization measurement taken from two samples won't have meaningful values if `scheduler_wall_time` was set to `false` (or the process that set it to true was terminated) between samples.

Reply:
In note box for sample/0 I wrote: "_If scheduler_wall_time has been disabled between two samples, passing them to utilization/2 will yield invalid results._".  But maybe that should be mentioned up in the description (also)?